fileglancer/settings.py

```diff
@@ -43,7 +43,7 @@ class Settings(BaseSettings):
     
     # The external URL of the proxy server for accessing proxied paths.
     # Maps to the /files/ end points of the fileglancer-central app.
-    external_proxy_url: Optional[HttpUrl] = None
+    external_proxy_url: HttpUrl = 'http://localhost:7878/files/'
```

Comment:
As far as I can tell, this makes the external_proxy_url "optional" by setting a default. If I leave it out, the server will start up and use the default value that is set here. Is that the functionality that we want? I was thinking that the server would require that this value be populated in the config.yaml or set as FGC_EXTERNAL_PROXY_URL. If either of those are missing, then the server should print a warning that the configuration is missing and shutdown.